### PR TITLE
Better car movement

### PR DESCRIPTION
**Pull request by iteadam**
Make the cars move smoother and the code look nicer

### CODE REVIEW

**Comment by iteadam** [comment]
There's some bug where the southern car goes off into the woods after delivery, very strange

File: packages/visualisation/src/App.js [hunk 1]
```diff
@@ -50,52 +54,94 @@ const App = () => {
...
+              } else { // change were it's going
+                currentCarWaypoints[event.carId] = { from: oldWaypoints.to, to: event }
+              }
+              break;
+            case 'car:pickup':
+              console.log('car is picking up package', event.bookingId)
+              break;
+            case 'car:deliver':
+              console.log('car is delivering package', event.bookingId)
+              // // console.log('event ======> ', event)
```
Comment (raduachim) [inline packages/visualisation/src/App.js:118]:
```suggestion
```

File: packages/visualisation/src/App.js [hunk 2]
```diff
@@ -50,52 +54,94 @@ const App = () => {
...
+              // console.log('bookings ===> ', bookings)
```
Comment (raduachim) [inline packages/visualisation/src/App.js:119]:
```suggestion
```

File: packages/visualisation/src/App.js [hunk 3]
```diff
@@ -50,52 +54,94 @@ const App = () => {
...
+              //debugger
```
Comment (raduachim) [inline packages/visualisation/src/App.js:120]:
```suggestion
```

**Review by mikaelaalu — APPROVED** [review]

**Pull request merged**